### ISSUE TITLE
Weird results on these objects

### ISSUE THREAD

**Issue by Zaxorn** [DLR-RM/AugmentedAutoencoder#87]
Hello, I read from your paper that AAE fails for long and thin objects, but I have strange results also in these types of objects.

![frame0_poseAAE](https://user-images.githubusercontent.com/58526402/94016334-43b08480-fdae-11ea-8688-949a2ae91695.png)
![frame0_poseAAE](https://user-images.githubusercontent.com/58526402/94016360-4c08bf80-fdae-11ea-8627-c6cfb4244349.png)

Is it possible to use AAE on these kind of objects or nothing? Thanks in advance

**Comment by MartinSmeyer** [comment]
Hi @Zaxorn ,
even though it can be hard, I have had better results on thin objects. Can you post the training_images (in a folder near your checkpoint)? It can happen that thin objects are not reconstructed well. In this case you can increase the  BOOTSTRAP_RATIO parameter in the config to something like 16.

**Comment by Zaxorn** [comment]
![training_images_39999](https://user-images.githubusercontent.com/58526402/94157753-df5ef500-fe81-11ea-833c-da65e57853eb.png)


Here is the training images setting the BOOTSTRAP_RATIO to 16 (I don't know why the embedding is so wrong).

This is the model.ply I am using for training but it looks like fine, like the others I have already succesfully used
[cifarelli_1.zip](https://github.com/DLR-RM/AugmentedAutoencoder/files/5276710/cifarelli_1.zip)

**Comment by MartinSmeyer** [comment]
after how many iterations? I suppose this is at the very beginning.

**Comment by MartinSmeyer** [comment]
You can also remove the occlusion augmentation to facilitate convergence. But it should work anyways.. What batch size did you use?

**Comment by Zaxorn** [comment]
50000 iterations with a batch size of 8.

Here is the cfg file used for training.
[chiave_candela.zip](https://github.com/DLR-RM/AugmentedAutoencoder/files/5276846/chiave_candela.zip)

**Comment by Zaxorn** [comment]
The picture is of the last iteration.

**Comment by MartinSmeyer** [comment]
You need to set the BOOTSTRAP_RATIO to 16 and retrain the networks from scratch. If you only use batch size 8 you might need to decrease the learning rate as well.

**Comment by Zaxorn** [comment]
Results look better on this object
![frame0_poseAAE](https://user-images.githubusercontent.com/58526402/94442646-80142400-01a4-11eb-91ca-311e442d4ec5.png)

but still wrong on the other one even if training looks good
![training_images_49999](https://user-images.githubusercontent.com/58526402/94442862-be114800-01a4-11eb-9381-c677eb0a56a7.png)

Do I need to further increase the BOOTSTRAP_RATIO? Anyway what does this parameter do?

**Comment by MartinSmeyer** [comment]
The bootstrap ratio refers to only calculating the reconstruction loss on the 1/bootstrap ratio pixels with the highest error. This forces the network not to just learn a constant background that is correct in most places. 

Yes, you can further increase this ratio, but not too much. As I said the occlusion can hide important features in the input here so commenting out this line should help
' CoarseDropout( p=0.2, size_percent=0.05) ),'
